webpay/pay/views.py

```diff
@@ -482,9 +482,10 @@ def _trim(st):
     # size unexpectedly.
     req['request']['description'] = _trim(req['request']['description'])
     if req['request'].get('locales'):
-        for k, v in req['request']['locales'].items():
-            d = _trim(v['description'])
-            req['request']['locales'][k]['description'] = d
+        for locale, data in req['request']['locales'].items():
```

Comment:
While the names are getting more meaningful... Perhaps `slug` or `name` and `locale` are better? I'm not crazy about `data` as a name and `locale` seems like what it is the data for. Easy enough to figure it out, just trying to avoid the generic `data` name.

```py
for slug, locale in req['request']['locales'].items():
    # ...